hazelcast/src/main/java/com/hazelcast/internal/config/MemberDomConfigProcessor.java

```diff
@@ -1736,21 +1707,20 @@ protected void handleMap(Node parentNode) throws Exception {
     void handleMapNode(Node parentNode, final MapConfig mapConfig) throws Exception {
         for (Node node : childElements(parentNode)) {
             String nodeName = cleanNodeName(node);
-            String value = getTextContent(node).trim();
             if (matches("backup-count", nodeName)) {
-                mapConfig.setBackupCount(getIntegerValue("backup-count", value));
+                mapConfig.setBackupCount(getIntegerValue("backup-count", getTextContent(node).trim()));
             } else if (matches("metadata-policy", nodeName)) {
-                mapConfig.setMetadataPolicy(MetadataPolicy.valueOf(upperCaseInternal(value)));
+                mapConfig.setMetadataPolicy(MetadataPolicy.valueOf(upperCaseInternal(getTextContent(node).trim())));
```

Comment:
Isn't the `.trim()` executed already in `getTextContent()` method?

Reply:
Yeah, it's a leftover. Will remove now